pyproject.toml

```diff
@@ -23,16 +23,22 @@ classifiers = [
 dependencies = [
     "absl-py==2.1.0",
     "attrs==23.2.0",
+    "cycler==0.12.1",
+    "fonttools==4.47.2",
+    "jupyter==1.0.0",
+    "kiwisolver==1.4.5",
     "matplotlib==3.8.2",
-    "mediapipe==0.10.10",
```

Comment:
Why to downgrade?

Reply:
Sorry, my mistake. Upgraded to the newer version == 0.10.11